content/docs/api-reference/dvcfilesystem.md

```diff
@@ -147,9 +162,25 @@ subdirectory.
 ['nlp/data/data.xml', 'nlp/data/features/test.pkl', 'nlp/data/features/train.pkl', 'nlp/data/prepared/test.tsv', 'nlp/data/prepared/train.tsv', 'nlp/eval/importance.png', 'nlp/model.pkl']
 ```
 
-## API Reference
+## fsspec API Reference
 
 As DVCFileSystem is based on [fsspec](https://filesystem-spec.readthedocs.io/),
-it is compatible with most of the APIs that it offers. For more details check
-out the fsspec's
+it is compatible with most of the APIs that it offers. When DVC is installed in
+the same Python environment as any other fsspec-compatible library,
```

Comment:
Maybe just mention hf datasets as an example here instead of leaving it cryptic?